### PR TITLE
Make raw result storage size configurable

### PR DESCRIPTION
**Pull request by JAORMX**
The ComplianceScan stores the raw reslults in a PVC. As of now, the size
was hardcoded to 1Gi. This introduces a parameter in the scan to be able
to set it. The aforementioned parameter is called
`rawResultStorageSize`, and it defaults to 1Gi.

### CODE REVIEW

File: pkg/controller/compliancescan/compliancescan_controller_test.go [hunk 1]
```diff
@@ -78,6 +80,52 @@ var _ = Describe("Testing compliancescan controller phases", func() {
...
+					Namespace: compliancescaninstance.Namespace,
+				}
+				reconciler.client.Get(context.TODO(), key, scan)
+				Expect(scan.Spec.RawResultStorageSize).To(Equal(compv1alpha1.DefaultRawStorageSize))
+			})
+		})
+
+		Context("With invalid RawResultStorageSize", func() {
+			It("report an error and move to phase DONE", func() {
+				compliancescaninstance.Spec.RawResultStorageSize = "invalid"
```
Comment (mrogers950) [inline pkg/controller/compliancescan/compliancescan_controller_test.go:114]: Would it be worth testing a scenario where you gave a valid size but one that the cluster can't accommodate due to other limits? i.e., 1000000Gi

Comment (JAORMX) [reply pkg/controller/compliancescan/compliancescan_controller_test.go:114]: That's a good point! Let me test it out locally and add a test case for that.

Comment (JAORMX) [reply pkg/controller/compliancescan/compliancescan_controller_test.go:114]: Should probably output an error too.

Comment (JAORMX) [reply pkg/controller/compliancescan/compliancescan_controller_test.go:114]: Done. Added an e2e test for that.

**Comment by JAORMX** [comment]
/retest

installer error

**Pull request merged**